packages/components/src/lib/queries/lexical-query.js

```diff
@@ -473,27 +397,20 @@ export default class LexicalQuery extends Query {
       })
     }
     Query.destroy(this)
-    return result
   }
 
-  getLexiconOptions (lexiconKey) {
+  static getLexiconOptions ({ lexiconKey, languageCode, location, siteOptions, resourceOptions }) {
```

Comment:
This function has been made static to eliminate dependency on the `LexicalQuery` instance. This way it can be used from the `Lexis` module without creating an instance of the `LexicalQuery`.